content/03.results.md

```diff
@@ -114,69 +129,74 @@ Table 1: Molecular subtypes determined across OpenPBTA samples.
-We summarized broad copy number and structural variation and observed that HGGs and DMGs, followed by medulloblastomas, have the most unstable genomes (**Figure {@fig:S3}A**).
-By contrast, craniopharyngiomas and schwannomas generally lack somatic copy number variation.
-These patterns of copy number variation largely align with estimates of tumor mutational burden (**Figure {@fig:S2}H**). 
-The number of SV and CNV breakpoints were significantly correlated across tumors (p = 1.08e-37) (**Figure {@fig:Fig3}C**) and as expected, the number of chromothripsis regions called increases as breakpoint density increases (**Figure {@fig:S3}B-C**).
-Chromothripsis events were observed in 41% (N = 19/46) of non-midline high-grade gliomas and 28.2% (N = 11/39) of DMGs (**Figure {@fig:Fig3}D**). 
-We also found evidence of chromothripsis in over 15% of embryonal tumors, ependymomas, meningiomas, germinomas, glial-neuronal tumors, chordomas, metastatic secondary tumors, and sarcomas, highlighting the genomic instability and complexity of pediatric brain tumors.
-We next assessed the contributions of eight previously identified adult CNS-specific mutational signatures [@doi:10.1038/s43018-020-0027-5] (RefSig) across cancer groups **Figure {@fig:Fig3}E** and samples **Figure {@fig:S4}A**.
-Stage 0 and/or 1 tumors characterized by low TMBs **Figure {@fig:S2}G** such as LGGs, gangliogliomas, craniopharyngiomas, DNETs, and schwannomas are expectedly dominated by Signature 1, which results from the normal process of spontaneous deamination of 5-methylcytosine.
-Signature N6 is CNS-specific signature which we observe nearly universally across samples.
-Drivers of Signature 18, _TP53_, _APC_, _NOTCH1_ (found at https://signal.mutationalsignatures.com/explore/referenceCancerSignature/31/drivers), are also canonical drivers of medulloblastoma, and indeed, we observe Signature 18 as the most common signature in medulloblastoma tumors.
-Signatures 3, 8, 18, and MMR2 are prevalent in HGGs, including DMGs.
-Finally, we observe that the weight of Signature 1 is higher at diagnosis (pre-treatment) and is almost always lower in tumors at later phases of therapy (progression, recurrence, post-mortem, secondary malignancy) **Figure {@fig:S4}B**.
-This trend may be the result of therapy-induced mutations which produce additional signatures (e.g., temozolomide treatment drives Signature 11), subclonal expansion, and/or acquisition of additional driver mutations during tumor progression, leading to higher overall TMBs and additional signatures.
+**TODO: A lot of different denominators here with no clear meaning. What is 666 vs 872 vs 54?**
+
+The top 50 mutated genes in primary tumors are shown **Figure {@fig:Fig3}** by tumor type (**A**, bar plots), with co-occurrence scores illustrated in the heatmap (**B**). **TODO: Clarify more what "top 50" means. Most mutations? Most commonly mutated across samples? etc.**
```

Comment:
@jashapiro can you speak to this TODO item regarding the specific meaning of "top 50"? Thanks!

Reply:
This is the set of genes that are mutated in the greatest number of samples. If a sample has more than one mutation in a gene, it is still only counted once.

Some more detail about what defines a countable mutation is at https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#recurrently-mutated-genes-and-co-occurrence-of-gene-mutations, though that should be updated to add that we exclude frequently mutated genes from non-tumor genomes, specifically the top 50 FLAGS genes (from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5706417/bin/12920_2017_309_MOESM3_ESM.txt, which is the corrected data for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4267152/)

Reply:
Ok, I'll just refer to the Methods here for an explanation of what top 50 refers to.